### PR TITLE
Fix url typo

### PR DESCRIPTION
**Pull request by KmilleF**

### CODE REVIEW

File: content/md/franklin-insights/franklin-insights-subscriptions.md [hunk 1]
```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
 
 You have 2 ways to subscribe products to Franklin:
```
Comment (NolwennP) [inline content/md/franklin-insights/franklin-insights-subscriptions.md]: I would remove the ":" if we are gonna remove the bullet points.

```suggestion
You have 2 ways to subscribe products to Franklin.
```

File: content/md/franklin-insights/franklin-insights-subscriptions.md [hunk 2]
```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
...
 
-- Via the **Product Edit Form**
+## Via the **Product Edit Form**
```
Comment (NolwennP) [inline content/md/franklin-insights/franklin-insights-subscriptions.md]:
```suggestion
## Via the Product Edit Form
```
No need for bold in titles.

File: content/md/franklin-insights/franklin-insights-subscriptions.md [hunk 3]
```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
...
 
 ![image](../img/Akeneo_Screenshot_Franklin_Insights_PEF_Subscription.png)
 
-- Via **bulk action**
+## Via **bulk action**
```
Comment (NolwennP) [inline content/md/franklin-insights/franklin-insights-subscriptions.md]:
```suggestion
## Via bulk action
```

Idem

File: content/md/franklin-insights/franklin-insights-subscriptions.md (hunk 1 above)
Comment (KmilleF) [reply content/md/franklin-insights/franklin-insights-subscriptions.md]: fixed

File: content/md/franklin-insights/franklin-insights-subscriptions.md (hunk 2 above)
Comment (KmilleF) [reply content/md/franklin-insights/franklin-insights-subscriptions.md]: fixed

File: content/md/franklin-insights/franklin-insights-subscriptions.md (hunk 3 above)
Comment (KmilleF) [reply content/md/franklin-insights/franklin-insights-subscriptions.md]: fixed

**Comment by KmilleF** [comment]
Duplicate: fixed in another PR

**Pull request closed without merging**